spaceflights/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipelines/data_processing/pipeline.py

```diff
@@ -26,3 +27,11 @@ def create_pipeline(**kwargs):
             ),
         ]
     )
+
+    namespaced_pipeline = pipeline(
+        pipe=pipeline_instance,
+        namespace="Data Processing",
```

Comment:
Does it behave well with empty spaces inside? I thought the keys in a yaml file can't have spaces, i.e. 

```yaml
Data Processsing.companies:
    type: parquet.ParquetDataSet
    ...
```

Reply:
it does kinda work - but I'll change it as it actually renders the same on viz